gcimagebundle/gcimagebundlelib/gentoo.py

```diff
@@ -0,0 +1,25 @@
+"""Gentoo specific platform info."""
```

Comment:
Also please add a disclaimer of the form @palladius indicated before I proceed with this merge, maybe right before the ```return True``` line. We're happy to have code that works on more OSes than we can provide official support for, but we do want that status to be clear to users to avoid confusion.